paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -145,13 +148,17 @@ def issue_state_change_for_service(service_config, force_bounce, desired_state):
 def paasta_start_or_stop(args, desired_state):
     """Requests a change of state to start or stop given branches of a service."""
     instance = args.instance
-    cluster = args.cluster
+    clusters = args.clusters
     soa_dir = args.soa_dir
     service = figure_out_service_name(args=args, soa_dir=soa_dir)
+    if args.clusters is not None:
+        clusters = args.clusters.split(",")
+    else:
+        clusters = list_clusters(service)
 
     service_config = get_instance_config(
         service=service,
-        cluster=cluster,
+        cluster=clusters[0],
```

Comment:
Should a bad service config halt execution altogether or should the command continue with any successful ones?